scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -214,6 +242,63 @@ class ScalametaParser(input: Input, dialect: Dialect) { parser =>
         i
       }
       val next = if (nextPos != -1) scannerTokens(nextPos) else null
+
+      // println(s"CURRENT ${curr.name} :: ${curr.text}")
+
+      // -1 if on token that doesn't start indent or whitespace,
+      // otherwise number of whitespaces till beginning of line for current token
+      lazy val currentIndent = {
+        def isWhitespace(token: Token): Boolean = token.is[Space] || token.is[Tab]
```

Comment:
https://dotty.epfl.ch/docs/reference/other-new-features/indentation.html#spaces-vs-tabs
we can have tabs and spaces. It is impossible to flatten tabs+spaces to single number, it depends on your editor what means a tab. For me it might be 2 spaces, for you maybe 4.
So far I treat tabs as 1 space, will get back to it far far later as it doesn't seem to be important